src/Cli/dotnet/dotnet.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageId>Microsoft.DotNet.Cli</PackageId>
     <UseAppHost>false</UseAppHost>
     <IsPackable>true</IsPackable>
+    <NoWarn>$(NoWarn);CS8002</NoWarn>
```

Comment:
Sprache isn't strong named. This is likely going to be an issue. @marcpopMSFT any pointers on how this should be handled?

Reply:
@leecow do we have any strong name signing requirements for .net core?

Reply:
Yes, I'm pretty sure sn signing is required across windows DLLs. @mmitche to confirm that we're doing this elsewhere.

Reply:
From @jkotas a bit back:

> Strong name signing does not matter in .NET Core. Nothing ever looks or verifies the signatures .NET Core, that includes crossgen. Ideally, we would be public-signing all .NET binaries that are not meant to run on desktop. https://github.com/dotnet/runtime/blob/main/docs/project/public-signing.md

Reply:
I've sidestepped the issue by using a hand-rolled parser here instead of Sprache.